src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        WorkWithFile workWithFile = new WorkWithFile();
+        workWithFile.getStatistic("banana.csv", "test.csv");
+        workWithFile.getStatistic("banana.csv", "test.csv");
```

Comment:
The `getStatistic` method is called twice with the same parameters. If this is not intentional for testing or demonstration purposes, consider removing the duplicate call to avoid redundancy.